core/federated/RTI/rti_remote.c

```diff
@@ -304,34 +333,33 @@ void handle_port_absent_message(federate_info_t* sending_federate, unsigned char
     // Need to acquire the mutex lock to ensure that the thread handling
     // messages coming from the socket connected to the destination does not
     // issue a TAG before this message has been forwarded.
-    lf_mutex_lock(&rti_mutex);
+    LF_MUTEX_LOCK(rti_mutex);
 
     // If the destination federate is no longer connected, issue a warning
     // and return.
-    federate_info_t* fed = GET_FED_INFO(federate_id);
+    federate_info_t *fed = GET_FED_INFO(federate_id);
     if (fed->enclave.state == NOT_CONNECTED) {
-        lf_mutex_unlock(&rti_mutex);
+        LF_MUTEX_UNLOCK(rti_mutex);
```

Comment:
Why was `LF_MUTEX_UNLOCK` introduced? I still see `lf_mutex_lock` and `lf_mutex_unlock` used in other places.

Reply:
I have open issue #327.